app/src/main/res/layout/patientsummary_bpplaceholderitem_content.xml

```diff
@@ -4,14 +4,13 @@
   android:id="@+id/placeHolderMessageTextView"
```

Comment:
I think layout file can also be removed since it was only being used in the old BP summary view.

Reply:
This layout is being used by `BloodSugarPlaceholderItemView` as well. That's why I left it.